fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/predicates/simplification/QueryPredicateComputationRule.java

```diff
@@ -23,7 +23,7 @@
 import com.apple.foundationdb.annotation.API;
 import com.apple.foundationdb.record.query.plan.cascades.matching.structure.BindingMatcher;
 import com.apple.foundationdb.record.query.plan.cascades.predicates.QueryPredicate;
-import org.apache.commons.lang3.tuple.Pair;
+import com.apple.foundationdb.record.util.pair.Pair;
```

Comment:
This is used as part of the type specification of the `RESULT` arg of an `AbstractQueryPredicateRule`. Given the way these rules are basically internal (and experimental), I just made them use the new `Pair`